lib/routes/account.js

```diff
@@ -646,6 +734,7 @@ module.exports = function (
             response.verificationMethod = 'email'
             response.verificationReason = 'login'
           }
+          recordSecurityEvent() // don't block response recording db event
```

Comment:
Ah I see, I think this actually affects the correctness of this. I noticed that `createResponse` returns early if the keys weren't wanted. What's the implications of this? Will that mean this session is never verified? Or that it is considered always verified? Something else?